### PR TITLE
docs: update consumer-group/router/service/script doc

### PR DESCRIPTION
**Pull request by hf400159**
### Description

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/en/latest/terminology/consumer-group.md [hunk 1]
```diff
@@ -64,7 +68,12 @@ curl http://127.0.0.1:9180/apisix/admin/consumers -H 'X-API-KEY: edd1c9f034335f1
 
 When APISIX can't find the Consumer Group with the `group_id`, the Admin API is terminated with a status code of `400`.
 
-If a Consumer already has the `plugins` field configured, the plugins in the Consumer Group will effectively be merged into it. The same plugin in the Consumer Group will not override the one configured directly in the Consumer.
+:::tip
+
+1. When the same plugin is configured in [consumer](./consumer.md), [routing](./route.md), [plugin config](./plugin-config.md) and [service] (./service.md), only one configuration is in effect, and the consumer has the highest priority. Please refer to [Plugin](./plugin.md).
```
Comment (SylviaBABY) [inline docs/en/latest/terminology/consumer-group.md:73]:
```suggestion
1. When the same plugin is configured in [consumer](./consumer.md), [routing](./route.md), [plugin config](./plugin-config.md) and [service](./service.md), only one configuration is in effect, and the consumer has the highest priority. Please refer to [Plugin](./plugin.md).
```

File: docs/zh/latest/terminology/router.md [hunk 2]
```diff
@@ -21,19 +26,25 @@ title: Router
 #
 -->
 
+## 描述
+
 APISIX 区别于其他 API 网关的一大特点是允许用户选择不同 Router 来更好匹配自由业务，在性能、自由之间做最适合选择。
 
-在本地配置 `conf/config.yaml` 中设置最符合自身业务需求的路由。
+你可以通过配置 `conf/config.yaml` 设置符合自身业务需求的路由。
```
Comment (SylviaBABY) [inline docs/zh/latest/terminology/router.md:33]:
```suggestion
你可以通过配置 `conf/config.yaml` 文件，来设置符合自身业务需求的路由。
```

**Review by SylviaBABY — APPROVED** [review]

File: docs/en/latest/terminology/service.md [hunk 3]
```diff
@@ -54,41 +58,52 @@ $ curl http://127.0.0.1:9180/apisix/admin/services/200 -H 'X-API-KEY: edd1c9f034
...
-    "service_id": "200",
-    "plugins": {
-        "limit-count": {
-            "count": 2000,
-            "time_window": 60,
-            "rejected_code": 503,
-            "key": "remote_addr"
+We can also specify different Plugins or Upstream for the Routes than the ones defined in the Service. The example below creates a Route with a `limit-count` Plugin. This Route will continue to use the other configurations defined in the Service (here, the Upstream configuration).
+
+    ```shell
```
Comment (juzhiyuan) [inline docs/en/latest/terminology/service.md:87]:
```suggestion
```shell
```

File: docs/zh/latest/terminology/consumer-group.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
+---
+title: Consumer Groups
```
Comment (juzhiyuan) [inline docs/zh/latest/terminology/consumer-group.md:2]: Consumer Group or Groups?

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**